localstripe/resources.py

```diff
@@ -885,7 +896,12 @@ def tax(self):
     def total_tax_amounts(self):
         concat = []
         for ii in self.lines._list:
-            concat.extend(ii.tax_amounts or [])
+            if ii.tax_rates:
+                concat.extend(ii.tax_amounts or [])
+            elif self.default_tax_rates:
+                concat.extend(
+                    [{'amount': int(ii.amount * tr.percentage / 100.0)}
```

Comment:
Any reason not to keep the format of https://github.com/adrienverge/localstripe/blob/8ef10f046e9d9d19359247547ffeefeb9dac59a8/localstripe/resources.py#L1294-L1296?

If real Stripe outputs `inclusive` and `tax_rate`, it's better to keep them. At the same time, you could canonize code with:
```python
class InvoiceItem(StripeObject):
    ...
    @property
    def tax_amounts(self):
        if self.tax_rates is not None:
            return [tr._tax_amount(self.amount) for tr in self.tax_rates]

class TaxRate(StripeObject):
    ...
    def _tax_amount(self, amount):
        return {'amount': int(amount * self.percentage / 100.0),
                'inclusive': self.inclusive,
                'tax_rate': self.id}
```